content/shared/influxdb3-admin/tables/create.md

```diff
@@ -0,0 +1,163 @@
+Use the [`influxdb3 create table` command](/influxdb3/version/reference/cli/influxdb3/create/table/)
+or the [HTTP API](/influxdb3/version/api/v3/) to create a table in a specified database in {{< product-name >}}.
+
+With {{< product-name >}}, tables and measurements are synonymous.
+Typically, tables are created automatically on write using the measurement name
+specified in line protocol written to InfluxDB.
+However, you can manually create tables to define a custom schema or apply custom settings before writing data.
+
+- [Create a table using the influxdb3 CLI](#create-a-table-using-the-influxdb3-cli)
+- [Create a table using the HTTP API](#create-a-table-using-the-http-api)
+- [Table naming restrictions](#table-naming-restrictions)
+
+## Create a table using the influxdb3 CLI
+
+1. If you haven't already, [download and install the `influxdb3` CLI](/influxdb3/version/reference/cli/influxdb3/#download-and-install-the-influxdb3-cli).
+
+2. Run the `influxdb3 create table` command and provide the following:
+
+   - _Required_: The name of the database to create the table in
+   - _Required_: The name of the table to create (see [Table naming restrictions](#table-naming-restrictions))
+   - _Required_: Tag columns to include in the table
+   - _Optional_: Field columns and their data types to include in the table
+   {{% show-in "enterprise" %}}
+   - _Optional_: A retention period for the table
+   {{% /show-in %}}
+
+   > [!Note]
+   > Tables must include at least one tag column.
+   > Field columns are optional and can be added later when you write data.
```

Comment:
We should also note the tag order and how it relates to this best practice: https://docs.influxdata.com/influxdb3/enterprise/write-data/best-practices/optimize-writes/#on-first-write-sort-tags-by-query-priority

The tag column order is important when you craet the table manually.

Reply:
9c14bddb3